utils/events/chat_helper_ticket_sys.py

```diff
@@ -587,6 +587,18 @@ def check(m):
                             reason="Ticket Perms",
                         )
                         RoleOBJ = discord.utils.get(guild.roles, name=role)
+                        
+                        if (
+                            RoleOBJ.id == MainID.r_chat_helper 
+                            or RoleOBJ.id == MainID.r_lead_helper
```

Comment:
Logic and everything looks good, although these ID's are outdated due to the server revamp. 
Could you go to core/common where `MainID.r_chat_helper` and `MainID.r_lead_helper` are and replace them with the following ids?

Lead Helper: 981764097936859146
Chat Helper: 978082630572597278

Reply:
Looks good!